src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private Node<K, V> [] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+        Node<K, V> node = new Node<>(key, value, null);
+        int indexOfTable = hash(key);
+        if (size == threshold) {
+            resize();
+        }
+        if (table[indexOfTable] == null) {
+            table[indexOfTable] = node;
+            size++;
+        } else {
+            Node<K, V> newNode = table[indexOfTable];
+            while (newNode != null) {
+                if (Objects.equals(node.key, newNode.key)) {
```

Comment:
You may use name of static method without it's class if you add static import for it
```suggestion
                if (equals(node.key, newNode.key)) {
```